test/index.test.js

```diff
@@ -277,6 +277,19 @@ describe('Google Analytics', function() {
           });
         });
 
+        it('should not set custom dimensions/metrics if settings.setAllMappedProps is false', function() {
```

Comment:
can you also write a test that this one gets fired properly?

`window.ga('send', 'pageview', pageview);`